src/main/groovy/com/blackducksoftware/integration/hub/detect/DetectConfiguration.groovy

```diff
@@ -165,6 +165,11 @@ class DetectConfiguration {
             }
         }
 
+        //you would only provide an alternate url for downloading the signature scanner if and only if you did not have a hub server
+        if (hubSignatureScannerHostUrl) {
+            hubOfflineMode = true
```

Comment:
What about a log with what you did only if you have to override it? 
if (... && hubOfflineMode == false) logger.info("A hub signature scanner url was provided, you probably meant to run in hub offline mode, setting hub offline mode to true")

Reply:
Ooo, I like it.